roles/grub/defaults/main.yml

```
@@ -0,0 +1,44 @@
+#
+# MIT License
+#
+# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+iso_url: "http://{{ server_name }}/hypervisor-x86_64.iso"
+kernel_path: /ephemeral/boot/x86_64/loader/kernel
+initrd_path: /ephemeral/boot/x86_64/loader/initrd.img.xz
+kernel_params:
+  - biosdevname=1
+  - console=tty0
+  - console=ttyS0,115200
+  - crashkernel=360M
+  - iommu=pt
+  - ip=dhcp
+  - neednet=1
```

```suggestion
  - rd.neednet=1
```

https://github.com/dracutdevs/dracut/blob/master/man/dracut.cmdline.7.asc#network